Formula/stellar-core.rb

```diff
@@ -2,9 +2,8 @@ class StellarCore < Formula
   desc "The backbone of the Stellar (XLM) network"
   homepage "https://www.stellar.org/"
   url "https://github.com/stellar/stellar-core.git",
-      :tag => "v9.2.0",
-      :revision => "7561c1d53366ec79b908de533726269e08474f77"
-  revision 1
+    :tag => "v10.0.0",
```

Comment:
Please don't change the indentation.

Reply:
Fixed in https://github.com/Homebrew/homebrew-core/commit/8bc52c37611e1a610f333287f626864e0d3babee